c-pallets/scheduler-credit/src/lib.rs

```diff
@@ -30,15 +30,18 @@ pub struct SchedulerCounterEntry {
 
 impl SchedulerCounterEntry {
 	pub fn increase_block_size(&mut self, block_size: u64) {
-		self.proceed_block_size += block_size;
+		self.proceed_block_size += block_size; // overflows?
```

Comment:
Checked has been used for calculation:
https://github.com/CESSProject/cess/pull/88/files#diff-d981858a8745d7f332607695d60b4906ea3f7dd08e693c264df4e9a691a8e1fcR33-R35